CHANGELOG.md

```diff
@@ -84,6 +84,8 @@
 * [ENHANCEMENT] Added `cortex_alertmanager_config_last_reload_successful_seconds` metric to show timestamp of last successful AM config reload. #3289
 * [ENHANCEMENT] Blocks storage: reduced number of bucket listing operations to list block content (applies to newly created blocks only). #3363
 * [ENHANCEMENT] Ruler: Include the tenant ID on the notifier logs. #3372
+* [ENHANCEMENT] Add flag `-<prefix>.tls-server-name` to require a specific server name instead of the hostname on the certificate. #3156
+* [ENHANCEMENT] Allow configuration of non-mutual TLS by using the flag `-<prefix>.tls-enabled`. #3156
```

Comment:
This looks a `[CHANGE]` to me, because TLS is no more implicitly enabled when other flags are set. You should mark it as a `[CHANGE]` and clarify it.

Reply:
Updated